packages/arb-token-bridge-ui/src/state/cctpState.ts

```diff
@@ -156,20 +156,14 @@ export const useCCTPDeposits = ({
   pageSize,
   enabled
 }: fetchCctpParams) => {
-  const { data, error, isLoading } = useSWRImmutable(
+  const { data, error, isLoading, mutate } = useSWRImmutable(
```

Comment:
We shouldn't be using `useSWRImmutable` if we're ultimately trying to mutate the data inside it. Shouldn't we simply user `useSWR`? 
https://swr.vercel.app/docs/mutation